pyproject.toml

```diff
@@ -33,3 +33,43 @@ skip = ["__init__.py"]
 
 [project.optional-dependencies]
 dev = ["documenteer[pipelines]"]
+
+
+[tool.towncrier]
+    package = "lsst.ts.externalscripts"
+    package_dir = "python"
+    filename = "doc/changes/CHANGES.rst"
+    directory = "doc/news"
+    filename_format = "{name}.{type}.rst|{name}.{type}.md"
+    title_format = "ts_externalscripts {version} ({project_date})"
+    issue_format = "`{issue} <https://jira.lsstcorp.org/browse/{issue}>`_"
+
+    [[tool.towncrier.type]]
+        directory = "feature"
+        name = "New Features"
+        showcontent = true
+
+    [[tool.towncrier.type]]
+        directory = "bugfix"
+        name = "Bug Fixes"
+        showcontent = true
+
+    [[tool.towncrier.type]]
+        directory = "perf"
+        name = "Performance Enhancement"
+        showcontent = true
+
+    [[tool.towncrier.type]]
+        directory = "doc"
+        name = "A Documentation Improvement"
```

Comment:
How about:

```
name = "Documentation"
```

?

Reply:
did you updated this one?

Reply:
Only in my mind. I forgot to save the changes. Now it is done.